src/Clients/user.py

```diff
@@ -17,8 +18,8 @@ class client():
 
     def __init__(self, client_id: str, image: image) -> None:
 
-        if (len(client_id) < 1):
-            raise ValueError("No value for Client ID has been passed")
+        if client_id == np.nan :
```

Comment:
As client_id is hinted as a String, will this catch values which are string representation of integers?

Reply:
yes it will catch

Reply:
I have been unable to get str, int or even python builtin None type to compare favorably to np.nan.
This seems unsuitable for a comparison. 

Commit needs to check if client_id is string representation of a number and is not None. This should be done using builtin None type of python

Reply:
if client_id == None, are you talking about this

Reply:
for checking string representation we use the ` is_int(client_id) ` method

Reply:
am I right? that's what you want should I make that changes or you want something else

Reply:
Yeah, we need to first make sure that the value is not none. 
Likewise, we can then verify using either a private function, or an internal check that the client ID is not a non-integer value, either using type checking or is instance functions.

for instance, if client_id is "abcd" then we need to raise an error. but not if it is "1234".

Reply:
Ok, I will make this change, wait for some time

Reply:
and if it is not a string representation then what?